create-release.py

```diff
@@ -609,7 +609,7 @@ def initialize_config(args=None) -> SimpleNamespace:
     configuration = {
         'goal': args.goal,
         'git_url': DEFAULT_GIT_URL,
-        'git_extension_package_url': DEFAULT_EXTENSION_PACKAGE_GIT_URL,
+        'git_branch': DEFAULT_GIT_BRANCH,
```

Comment:
```suggestion
        'git_branch': args.branch or DEFAULT_GIT_BRANCH,
```

Reply:
Lets add a branch option to the cmd line instead of setting envs prior to running
```
parser.add_argument('--branch', help='the branch name to use', type=str, required=False)
```